indico/modules/events/editing/service.py

```diff
@@ -148,3 +150,25 @@ def service_handle_new_editable(editable):
     except requests.RequestException as exc:
         logger.exception('Failed calling listener for editable')
         raise ServiceRequestFailed(exc)
+
+
+def service_handle_review_editable(editable, revision):
+
+    class PublishSchema(mm.Schema):
+        publish = fields.Boolean()
+        success = fields.Boolean()  # TODO: We can probably remove this, or find a simpler alternative
```

Comment:
I don't think we really need a schema for this, do we? Just using the dict coming from the json response is probably more than enough...

Reply:
I agree, I was debating about the same thing.